ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -30,5 +30,281 @@
 #include <catch2/catch.hpp>
 
 #include "azure_connection.h"
+#include <fstream>
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("Testing Azure Connection Class", "azure-connection")
+{
+
+    Azure::Storage::Blobs::BlobContainerClient azure_client =
+      Azure::Storage::Blobs::BlobContainerClient::CreateFromConnectionString(
+        std::getenv("AZURE_STORAGE_CONNECTION_STRING"), "myblobcontainer1");
+    bool exists = false;
+
+    const std::string object_name = "test_object";
```

Comment:
Can we create a function that creates the file system, it will look alot cleaner.

Reply:
yes this would make things a lot cleaner but this also means that we are stuck with the same state when we start each test which may not be optimal. In addition it is set out this way because a lot of the variables are used in the test cases below.

Reply:
created in https://github.com/wiredtiger/wiredtiger/pull/8747/commits/195842c01a12929f8a1e30bbf9f0f6522bf03cff